helix-term/src/commands.rs

```diff
@@ -95,6 +95,11 @@ impl<'a> Context<'a> {
         });
         self.callbacks.push(callback);
     }
+
+    #[inline]
+    pub fn get_count_or_one(&self) -> usize {
```

Comment:
maybe just call this `count()` and name the variable `_count`? Or `get_count()`

Reply:
I agree on `count()`.

Reply:
updated.